front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -141,7 +141,6 @@
 
     .scenario-description {
       position: relative;
-      text-wrap: wrap;
```

Comment:
Wasn't this important for very long words without spaces in them? (and stuff like URLs etc)

Reply:
Normally not, because I used a `word-break: break-all;` for the description.